### ISSUE TITLE
CMDer won't open Sublime

### ISSUE THREAD

**Issue by buildersbrewery** [cmderdev/cmder#814]
Running `cmder v1.2.9` with `Sublime Text 3 Dev Build 3095` with:

```batch
subl="%CMDER_ROOT%\vendor\sublime\sublime_text.exe" $1
```

doesn't work. Using something like:

```batch
subl="D:\PortableApps\PortableApps\cmder\vendor\sublime\sublime_text.exe" $1
```

instead works fine. How can I fix this so it'll work for me when using my USB drive on another PC where the USB drive will be assigned something else rather than `D:\...`?

**Comment by JanSchulz** [comment]
This is probably https://github.com/cmderdev/cmder/issues/791 -> should be gone in one of the newer dev builds

**Comment by buildersbrewery** [comment]
@JanSchulz Seems like it. I can confirm it's on Win10.

**Comment by buildersbrewery** [comment]
fixed by manually downloading clink v0.4.7
